### PR TITLE
Added rendering settings for processed images

### PR DESCRIPTION
**Pull request by dominikl**
See description.

### CODE REVIEW

**Comment by dominikl** [comment]
`yamllint` is not happy. But it's valid formatting and could be applied without problems. That's the format which is printed out by the cli render plugin. If we want to fix that, we should make the output of the render plugin yamllint compatible.

File: experimentA/Exp2_rep3_projection.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+channels:
+  0: {active: true, color: FFFFFF, end: 1.0, label: Cells, max: 1.0, min: 0.0, start: 0.0}
```
Comment (sbesson) [inline experimentA/Exp2_rep3_projection.yml]: Just realized these are 0-indexed while the input format for the `render` plugin should be 1-based?

**Comment by sbesson** [comment]
@dominikl last two commit include some additional code to reformat the YAML in a `yamllint` compliant style. Happy to merge this as it is if you don't have anything else.

Main question is whether the style options in ad6bf78 should be ported back to `render info` so that the generated YML is compliant from the start.

**Comment by dominikl** [comment]
Definitely `render info` should produce output without yamllint warnings/errors. I'll have a look / open a PR.

**Pull request merged**